misc/cargo-vet/audits.toml

```diff
@@ -1513,7 +1483,7 @@ delta = "6.0.6 -> 7.0.0"
 
 [[trusted.abomonation]]
 criteria = "safe-to-deploy"
-user-id = 704
+user-id = 704 # Frank McSherry (frankmcsherry)
```

Comment:
Are we still using abomonation for anything load bearing, and how can we fix that?

Reply:
Nope! https://github.com/MaterializeInc/materialize/pull/30264 to remove it.